tests/molecular/molecules/building_block/fixtures/init_from_rdkit_mol.py

```diff
@@ -82,6 +86,14 @@
             core_atom_ids=(0, 1, 2, 3),
             placer_ids=(0, 1, 2, 3),
         ),
+        CaseData(
+            building_block=stk.BuildingBlock.init_from_rdkit_mol(
+                molecule=dative_molecule,
+            ),
+            functional_groups=(),
+            core_atom_ids=(0, 1),
+            placer_ids=(0, 1),
+        ),
```

Comment:
I think the test case for dative bonds needs to go into the molecule case data, because thats that case data that get tested by get_bonds() -> which is what we want to test